src/components/PublishContainer/PublishedScreen/PublishedScreen.tsx

```diff
@@ -42,6 +44,19 @@ export const PublishedScreen: FunctionComponent<PublishScreen> = ({
     };
   });
 
+  const generateZipFile = (): void => {
+    const zip = new JSZip();
+    publishedDocuments.forEach((document) => {
+      const file = JSON.stringify(document.wrappedDocument);
```

Comment:
if u want pretty json:

```suggestion
      const file = JSON.stringify(document.wrappedDocument, null, 2);
```

Reply:
ok